src/scenes/home/gala/gala.js

```diff
@@ -1,119 +1,58 @@
 import React from 'react';
-import Section from 'shared/components/section/section';
+import ReactRevealText from 'react-reveal-text';
+import commonUrl from 'shared/constants/commonLinks';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import Styles from './gala.css';
```

Comment:
Since it's not a component, 'Styles' should be lowercase here, that's the pattern used elsewhere in the app.